### PR TITLE
Fix footer overlapping three dot menu for tags

### LINKED ISSUE #9271: Footer overlaps tag note & three dot menu
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?
When accessing a tag, the footer overlaps the three dots menu when it is open. 
This hides some of the options on the three dots menu behind the footer.
Screenshot 2021-03-05 at 2 30 40 PM

> What did you expect to see that you didn't?
I expected the three dots menu to appear in front of the footer, a margin below the tag could also separate the footer from covering the three dots menu when it is open.

### Please show us where to look

https://publiclab.org/questions/tag/lat:5

### What's your PublicLab.org username?

>manchere

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!
Google chrome Version 88.0.4324 & Safari Version 14.0.2

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by manchere**
Fixes #9271 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### Screenshots of changes in the UI

- Old UI

Screenshot 2021-03-05 at 2 32 17 PM

- New UI

Screenshot 2021-03-13 at 3 25 58 PM

### CODE REVIEW

**Comment by manchere** [comment]
I would like a review please @pydevsg

**Review by pydevsg — COMMENTED** [review]
Looks good to me @manchere. Can you please restart the Gitpod, once?
It's failing in functional-tests (pull_request)

**Comment by cesswairimu** [comment]
I will try to close and reopen this for rebuild..thanks

File: app/views/tag/show.html.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@
   background-size: cover;
   background-color: #FFF;
 }
-.fdb-block, .rounded {
+.rounded {
```
Comment (jywarren) [inline app/views/tag/show.html.erb:37]: Ah, i'm a little worried what happens when this has a background image now. Could you go to the corresponding wiki page (or create one if you need, where `/tag/tagname` corresponds to `/wiki/tagname`), and see if a tall image then goes out of the frame? If so, we may need to think of another way around this somehow...

Thanks for catching this issue though!!!!

Comment (manchere) [reply app/views/tag/show.html.erb:37]: @jywarren I am sorry I am facing connection issues in my localhost development probably caused by rvm or redis...  I think it is causing other problems for me to make a first post and bypass the moderation system. Is there any place I can address my issue?

Comment (cesswairimu) [reply app/views/tag/show.html.erb:37]: Hi @manchere, do you have a specific error that you are getting?  we could try and debug

Comment (manchere) [reply app/views/tag/show.html.erb:37]: @cesswairimu I would love to, thank you.

Comment (cesswairimu) [reply app/views/tag/show.html.erb:37]: feel free to post the error you are getting here or browse through #3840 you may find a fix for it over there. Thanks

**Comment by Tlazypanda** [comment]
Hey @manchere Just checking in here please feel free to ping incase you have any questions or need any help :tada:

**Comment by manchere** [comment]
This is the issue I am currently facing:
An error occurred while installing mimemagic (0.3.10), and Bundler cannot continue.
Make sure that `gem install mimemagic -v '0.3.10' --source 'https://rubygems.org/'` succeeds before bundling.

**Comment by cesswairimu** [comment]
Hey @manchere  I see a comment in my mail but can't trace it here, if you are still getting the mimemagic error please pull our latest code and bundle install, you should be good. thanks

**Comment by manchere** [comment]
Hey @cesswairimu thank you so much I pulled the latest code and it solved the mimemagic error, I am currently facing an error with my webpacker  'Webpacker can't find application.js' and my manifest file is empty

**Comment by cesswairimu** [comment]
Did you ran this during installation?  `rails webpacker:install && rails webpacker:install:react && rails generate react:install`@manchere ...could be you installed the project before we made the improvements...also you could try and use gitpod to make the changes remotely if you are still having trouble with that as we figure it out https://gitpod.io/#https://github.com/publiclab/plots2/pull/9299 . thanks

**Comment by Priyaraj17** [comment]
May I review this pull request?

**Pull request closed without merging**